### PR TITLE
Pia 1807 german translation

### PR DESCRIPTION
**Pull request by a-szotyori**
Also contains a small fix regarding input field error messages. When an input field has an error then on focus the error message is only hidden. It is resumed after loosing focus. This way the user can't clear error messages just by switching focus between input fields.

### CODE REVIEW

**Comment by a-szotyori** [comment]
Keeping it as a draft until the German translations have been verified.

File: health-sdk/sdk/src/main/res/values-en/strings.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<resources>
+    <string name="ghs_amount_hint">Amount</string>
+    <string name="ghs_purpose_hint">Purpose</string>
+    <string name="ghs_recipient_hint">Recipient</string>
+    <string name="ghs_iban_hint">IBAN</string>
+    <string name="ghs_pay_button">Pay</string>
+    <string name="ghs_bank_placeholder">Loading...</string>
+    <string name="ghs_error_failed_to_load_image">Failed to load image</string>
+    <string name="ghs_error_input_invalid_iban">Invalid IBAN</string>
+    <string name="ghs_error_input_invalid_Currency">Invalid currency</string>
```
Comment (zladzeyka) [inline health-sdk/sdk/src/main/res/values-en/strings.xml:11]: When is an invalid currency error happen?

**Review by zladzeyka — APPROVED** [review]
👏🏻

**Review by zladzeyka — APPROVED** [review]
Great 😊 job!

**Pull request merged**